crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -81,7 +83,11 @@ pub struct IatapayPaymentsRequest {
 impl TryFrom<&types::PaymentsAuthorizeRouterData> for IatapayPaymentsRequest {
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
-        let country = item.get_billing_country()?.to_string();
+        let payment_method = item.payment_method;
+        let country = match payment_method {
+            PaymentMethod::Upi => "IN".to_string(),
```

Comment:
Hardcoding is not the right way. Can you create an issue to tackle this later.

Reply:
Hey, this is merchant requirement to hardcode country as UPI is only applicable in India. cc: @shakthi-22